docs/remote/troubleshooting.md

```diff
@@ -17,6 +17,12 @@ For tips and questions about [GitHub Codespaces](https://github.com/features/cod
 
 SSH is powerful and flexible, but this also adds some setup complexity. This section includes some tips and tricks for getting the Remote - SSH extension up and running in different environments.
 
+## Customize AI Chat Responses
+
+[Custom instructions](/docs/copilot/copilot-customization.md) enable you to describe common guidelines or rules to get responses that match your specific coding practices and tech stack.
+
```

Comment:
Link to the Dev Containers section mentioning that this has more details for Dev Containers?

Reply:
Good call out! Just updated.